Document applications & teams (#145)
discord-userdoccers/discord-userdoccers: pages/resources/application.mdx

| guild_id?                                    | snowflake                                                               | The ID of the guild linked to the application                                                                                                                                        |
| eula_id?                                     | snowflake                                                               | The ID of the [EULA](/resources/store#eula-object) required to play the application's game                                                                                           |
| slug? ^1^                                    | string                                                                  | The URL slug that links to the primary store page of the application                                                                                                                 |

> | aliases?                                     | array[string]                                                           | Other names the application's game is recognized by                                                                                                                                  |

Reviewer: ```suggestion
| aliases?                                     | array[string]                                                           | Other names the application's game is associated with                                                                                                                                  |
```

| interactions_endpoint_url                    | string                                                                  | The URL of the application's [interactions endpoint](/interactions/receiving-and-responding#receiving-an-interaction)                                                                |
| interactions_version                         | integer                                                                 | The [version of the application's interactions endpoint implementation](#application-interatcions-version)                                                                           |
| interactions_event_types ^4^                 | array[string]                                                           | The enabled [Gateway events](/topics/gateway-events) to send to the interaction endpoint                                                                                             |

> | tags?                                        | array[string]                                                           | Tags describing the content and functionality of the application (max 20 characters, max 5)                                                                                          |

Reviewer: ```suggestion
| tags?                                        | array[string]                                                           | Tags describing the content and functionality of the application, processed to categorise the app in the App Directory (max 20 characters, max 5)                                                                                          |
```

Author: I don't think this is only for app directory

| install_params?                          | ?[application install params](#application-install-params-object) object | The default in-app authorization link for the integration                                                                                                                                |
| custom_install_url?                      | ?string                                                                  | The default custom authorization link for the integration                                                                                                                                |
| discoverability_state?                   | integer                                                                  | The current application directory [discoverability state](#application-discoverability-state) of the application (only `NOT_DISCOVERABLE` and `DISCOVERABLE` is supported)               |

> | max_participants?                        | ?integer                                                                 | The maximum number of people that can participate in the application's embedded activity (-1 for no limit)                                                                               |

Reviewer: ```suggestion
| max_participants?                        | ?integer                                                                 | The maximum possible participants in the application's embedded activity (-1 for no limit)                                                                               |
```
